src/cljs/proton/layers/core/keybindings.cljs

```diff
@@ -1,10 +1,31 @@
 (ns proton.layers.core.keybindings
   (:require [proton.layers.core.actions :as actions :refer [state]]
+            [proton.lib.pane_manager :as panes]
             [proton.lib.proton :as proton]))
 
+(defn select-window-fn [n]
```

Comment:
Do we need this to be its own function, or could we jsut use `panes/focus-on-item` in place of `select-window-fn` in the keybindings map?